lib/resty/upstream/healthcheck.lua

```diff
@@ -597,6 +605,16 @@ function _M.spawn_checker(opts)
         return nil, "no upstream specified"
     end
 
+    local hook_down = opts.hook_down
+    if (hook_down and type(hook_down) ~= 'function') then
```

Comment:
- I think the lua-resty codestyle uses double quotes instead of singles.
- idem: parenthesis?